blueoil/blueoil_init.py

```diff
@@ -101,6 +102,14 @@
 # ]
 
 
+learning_rate_schedule_map = OrderedDict({
+    "constant": "'constant' -> constant learning rate.",
+    "2-step-decay": "'2-step-decay' -> learning rate reduce to 1/10 on epochs/2 and epochs-1.",
```

Comment:
I think `epochs` is ambiguous.

Reply:
I think `learning rate reduce to 1/10` means lr is changed to 0.1.
It may be better to change to `learning rate is reduce by 1/10`.

Reply:
Maybe `number of epochs` is popular.
How about this?

Reply:
Sure!!

Reply:
@lm-konda
but I already changed messages using `{epochs}`
https://github.com/blue-oil/blueoil/blob/ea8f63aa5b530401a43f9002c4ffa0fc7ee9a4d4/blueoil/blueoil_init.py#L308

‘number of epochs’ is better than `{epochs}` ?

Reply:
I think there is no change from the state that the meaning is ambiguous in essence.

Reply:
@lm-konda 
Is the below message ambiguous for you (and user)?
https://github.com/blue-oil/blueoil/blob/ea8f63aa5b530401a43f9002c4ffa0fc7ee9a4d4/blueoil/blueoil_init.py#L308

Reply:
I was worried about how to change, but now I feel good even as it is.